dashboard/lib/pd/foorm/helper.rb

```diff
@@ -24,12 +24,35 @@ def get_survey_key(ws_submission)
     def get_index_for_survey_key(survey_key)
       if survey_key == "Pre Workshop"
         return 0
+      elsif survey_key.start_with? "Pre Workshop - Module"
+        # last character will be the module number, use that as an index
+        return survey_key[survey_key.length - 1].to_i
+      elsif survey_key.start_with? "Pre Workshop"
+        # either in person or unknown agenda, we want it to be before daily results
+        return 1
+      elsif survey_key.include?("Day")
+        return 100 + survey_key[4..-1].to_i
+      elsif survey_key == "Post Workshop"
+        return 1000
+      elsif survey_key.starts_with? "Post Workshop - Module"
+        # last character will be the module number, use that as an index
+        return 1000 + survey_key[survey_key.length - 1].to_i
+      else
```

Comment:
What does the else represent here? Won't this be the same value as post-workshop for module 1?

Reply:
this would be the case of in_person agenda, which I should just combine with "Post Workshop", since both would not exist at the same time. Module 1 and in person would never exist for the same workshop to your question specifically.